mock/src/components/mockedJson.ts

```
@@ -0,0 +1,149 @@
+// Map containing example CSV data where the keys represent file names
+// and the values represent objects with 'data' and 'hasHeaders' properties
+export const mockedDataMap = new Map<
+  string,
+  { data: string[][]; hasHeaders: boolean }
```

awesome extensive mock data. I like how you even have error messages returned!